### ISSUE TITLE
Show all public space in MY SPACE

### ISSUE THREAD

**Issue by Grisu1959** [humhub/humhub#4415]
Hello
I set different pubblic space (space where a user can ask to go inside)
but in MY SPACE are shown just a few
I would like have to possibility to show there all the aviable space (as in the menù  LISTS->SPACE)
and a user can choose if join or not

**Comment by buddh4** [comment]
Yes there should be a better way especially for new members to search/find new spaces. But I'am not sure about adding
non related spaces to the space chooser. Perhaps rather add a button/link to the space chooser like "Search spaces" or "Overview"

A better space overview/search page would be nice something like:

 - My space memberships
 - Spaces I follow
 - New Spaces
 - Recommendations
 - Search

Another enhancement in this direction would be a notification if a new public space was created.

**Comment by Carl2213** [comment]
Add a "show all spaces" link to the bottom of the space chooser which goes to the space directory.

Recommendations would be nice, but a bit unnecessary I think. Spaces of my friends would be a simpler alternative that is still pretty useful.
